README.md

```diff
@@ -34,7 +34,7 @@ ssb.tribes.create({}, (err, info) => {
     test: 'kia ora, e te whānau',
     recps: [groupId] // <<< you can now put a groupId in the recps
   }
-  ssb.publish(content, (err, msg) => {
+  ssb.tribes.publish(content, (err, msg) => {
```

Comment:
you should be able to change to `ssb.db.create`. does that work?

Reply:
that works so far but i want to include tangle-attaching in tribes.publish